cogs/canvas.py

```diff
@@ -501,6 +504,9 @@ def canvas_init(self):
             guild = self.bot.guilds[[guild.id for guild in self.bot.guilds].index(int(c_handler_guild_id))]
 
             if guild not in (ch.guild for ch in self.bot.d_handler.canvas_handlers):
+                # Note to developers
+                # If you're testing this bot on your own server, replace "" with CANVAS_API_KEY
+                # When pushing the code to GitHub, replace CANVAS_API_KEY with "".
```

Comment:
What do you think of adding something along the lines of 
`# CANVAS_API_KEY always required for non-public access courses`?
It would make it clearer for future contributors to realise why this is empty right now.

Reply:
That's a good idea.